ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-commands.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
Why do we need a separate step for this (and not use the common openshift-e2e-test)? The script does not seem to refer to any libvirt/multiarch code.

Reply:
> Why do we need a separate step for this (and not use the common openshift-e2e-test)? The script does not seem to refer to any libvirt/multiarch code.

The libvirt requires dnsConfig in ref to connect to the cluster.

Reply:
I think we might be the only ones who use an exclude list - like in the `suite` function.
That said, I don't know if that could be simply consolidated into the common code.

Reply:
In case of libvirt/multiarch we need pod dnsConfig to point towards coredns pod to resolve dns requires pointing towards cluster inside yellow zone. To consolidate we need to update the pod dnsConfig for the step which according to me is not a great idea affecting all the jobs using common  step openshift-e2e-test.